### ISSUE TITLE
1.2.0 fails with "image not found"

### ISSUE THREAD

**Issue by jonas-eschle** [sphinx-contrib/youtube#36]
Thanks a lot for this neat addon!

With 1.2.0, our docs would fails with
```
sphinx.errors.ExtensionError: Handler <function download_images at 0x7f4bf7164830> for event 'env-updated' threw an exception (exception: [Errno 2] No such file or directory: '/home/docs/checkouts/readthedocs.org/user_builds/zfit/checkouts/412/docs/_video_thumbnail/nGzyXVXfHnw.jpg')
```
where nGzyXVXfHnw is the youtube video code in the directive as

```
..  youtube:: nGzyXVXfHnw
       :width: 320
```

The whole build and failure can be [found here](https://readthedocs.org/projects/zfit/builds/17391075/).

With 1.1 it works

**Comment by rlaphoenix** [comment]
See https://github.com/sphinx-contrib/images/pull/31
Minimal reproducible example: https://github.com/rlaphoenix/rtd-dir-reproducible-example

This issue at it's core seems to be related to the usage of sphinxcontrib-youtube v1.2.0 with spinxcontrib-images

**Comment by 12rambau** [comment]
let me know but normally I've solved this one in #40

**Comment by jonas-eschle** [comment]
seems solved to me, closing
